src/Sulu/Bundle/WebsiteBundle/Sitemap/SitemapContentQueryBuilder.php

```diff
@@ -29,10 +29,10 @@ protected function buildWhere($webspaceKey, $locale)
                 ISDESCENDANTNODE('/cmf/%s/contents')
                 OR ISSAMENODE('/cmf/%s/contents')
             ) AND (
-                page.[i18n:de-seo-hideInSitemap] IS NULL
-                OR page.[i18n:de-seo-hideInSitemap] = false
+                page.[i18n:%s-seo-hideInSitemap] IS NULL
+                OR page.[i18n:%s-seo-hideInSitemap] = false
             )
-        )", $webspaceKey, $webspaceKey, $locale);
+        )", $webspaceKey, $webspaceKey, $locale, $locale, $locale);
```

Comment:
if I see this correctly there are only 4 `%s` so I think it only need 1 additional $locale.